README.md

```diff
@@ -134,8 +134,9 @@ Follow these steps to automatically instrument your application using the Java (
 
     Insert the `-javaagent` flag before the `-jar` file, adding it as a JVM option, not as an application argument.
 
-> To generate a snippet that includes all the basic install commands for your environment and service, open the Observability Cloud
-wizard in **Data Setup > APM Instrumentation > Java > Add Connection**.
+> **Note**: Supported java servers may require different steps to add the path to the JVM agent to their configuration. See [Add the JVM agent to Java servers](./docs/server-instructions.md).
+
+> **Tip**: To generate a snippet that includes all the basic install commands for your environment and service, open the Observability Cloud wizard in **Data Setup > APM Instrumentation > Java > Add Connection**.
```

Comment:
maybe add some verbiage that this is a thing in the Splunk O11y UI? If I didn't know this was a thing, I might be confused about what you're referring to with this (valuable) tip.

Reply:
Or can we deep link to it?

Reply:
if possible, that would be even betrer.

Reply:
Not possible, unfortunately: each client has its own subdomain / realm, afaik.